Packs/FeedFeodoTracker/Integrations/FeedFeodoTrackerIPBlocklist/FeedFeodoTrackerIPBlocklist.py

```diff
@@ -0,0 +1,66 @@
+from CommonServerPython import *
+
+
+def main():
+    params = {k: v for k, v in demisto.params().items() if v is not None}
+    chosen_urls = []
+    params['feed_url_to_config'] = {}
+    sources = params['feed_source']
+    if 'Last 30 Days' in sources:
```

Comment:
not sure that those names are the right ones, need to double check with Ryan here

Reply:
will send him a message